spec/features/user_visits_place_spec.rb

```diff
@@ -56,4 +74,22 @@
       end
     end
   end
+
+  describe "that doesn't exist" do
+
+    it 'renders a SORRY page' do
+      VCR.use_cassette("nonexistnat place") do
+        visit '/'
+
+        page.fill_in 'nav-search',
+          :with => 'askljasdf'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.